### ISSUE TITLE
Accessibility: announce the group name of an editor

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17230]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: NVDA need to be opened.
1.Open Visual studio code editor.
2.Using "Ctrl+1" key navigate to the left editor.
3.Using "Ctrl+2" key navigate to the side editor. 
4.Using "Ctrl+3",key navigate to the Right editor.

**Expected Result:**
NVDA should announce the name of the editor group  while navigating to 
Ex: NVDA should announce "app.js text editor 1" when the user is navigated to first editor.

**Actual result:**
NVDA is not reciting the name for the editor groups while navigating to it.

**User Impact:**
Screen Reader user doesn't get to know the which Editor they are in and on pressing "Ctrl+1" they are navigated to the editor or not is not known.

![inspectinformation_visualstudiocodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212628/91b9ae42-c2b4-11e6-9837-8ee4c6817a4a.jpg)
![narratorbuddyinformation_vscodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212630/9550e714-c2b4-11e6-8631-fab6cf721998.jpg)
![nvda_speechviewerinformation_vscodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212633/97f456fe-c2b4-11e6-8e3d-d8130303a1df.jpg)

**Comment by WiproPSAccTest** [comment]
@bpasero  Here NVDA is not informing the blind user in which editor he/she has moved to. It is simply announcing the name of the open file only for that editor.

**Comment by bpasero** [comment]
@WiproPSAccTest I fixed it, you can verify from our insider release. 
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: http://code.visualstudio.com/Download#insiders

**Comment by WiproPSAccTest** [comment]
@bpasero I checked the issue in vs code editor Insider Build but NVDA is not providing proper information about the editors. For both 2nd and 3rd editor it is reciting the same information.

**Comment by bpasero** [comment]
Thanks, I can reproduce.

**Comment by bpasero** [comment]
Will push a fix that reads the position of the editor as part of the ARIA label (1, 2, or 3).

**Comment by WiproPSAccTest** [comment]
Title has been changed to a common format to perform data population.
